docs/wiki/Fetching latest changes - Scenario 2 Module library and CI environment.md

```diff
@@ -0,0 +1,55 @@
+In this scenario you have adopted the [Pipeline Orchestration](./Solution%20creation#pipeline-orchestration) approach and you have onboarded the module library and the CI environment, as described in [Getting started Scenario 2 - Onboard module library and CI environment](./Getting%20started%20-%20Scenario%201%20Onboard%20module%20library%20and%20CI%20environment).
```

Comment:
I wouldn't map scenario 1 and 2 to respectively template and pipeline orchestration. With template vs pipeline orchestration, we mainly refer to how multiple modules can be combined together to build a solution. I'd suggest to remove this reference, something like:
```suggestion
In this scenario, you have onboarded both the module library and the CI environment, as described in [Getting started Scenario 2 - Onboard module library and CI environment](./Getting%20started%20-%20Scenario%201%20Onboard%20module%20library%20and%20CI%20environment) and you would therefore need to fetch latest changes for both.
```

Reply:
Thank you Erika, i'm updating it